MongoDB/containers/0.1.0/mongodb-config/entrypoint.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Installing dnsutils and jq
+echo "deb http://http.debian.net/debian wheezy-backports main" | tee /etc/apt/sources.list.d/wheezy-backports.list > /dev/null
+echo "deb http://security.debian.org/ wheezy/updates main contrib non-free " | tee /etc/apt/sources.list.d/wheezy-security.list > /dev/null
+echo "deb-src http://security.debian.org/ wheezy/updates main contrib non-free" | tee /etc/apt/sources.list.d/wheezy-security.list > /dev/null
+apt-get -q update > /dev/null
+apt-get install -qqy dnsutils jq > /dev/null 2>&1
+
+# Check for lowest ID
+/opt/rancher/bin/lowest_idx.sh
```

Comment:
I get why this script is installing dig and jq here, but would it be possible to install dig and jq into the alpine container and pass the result via file to the entrypoint? It looks like the dig output goes to file in the other script. The lowest_idx.sh could be changed to write a file instead of looking at exit codes maybe. I could be missing something here, just wanted to see if we could avoid installing at runtime. Could bite us in the future as new versions roll out.